modules/date-localizer/test/spec.coffee

```diff
@@ -3,79 +3,221 @@ describe 'dateTimeLocalizerMoment', ->
 
 describe 'dateTimeLocalizer', ->
   localizer = hx.dateTimeLocalizer()
-
+  zeroPad = hx.format.zeroPad(2)
   testDate = new Date(1452130200000) # Thu Jan 07 2016 01:30:00 GMT+0000 (GMT)
   zeroHourDate = new Date(1452124800000) # Thu Jan 07 2016 00:00:00 GMT+0000 (GMT)
   invalidDate = new Date('Invalid Date')
+  # 0 - London, -120 - Sofia, -60 Krakow, -540 Tokio, 420 Pinal (Arizona/United states)
+  timezonesOfsets = [0, -120, -60, -540, 420]
+
+  getHexaGonTimeZoneOffset = ->
+    hexagonTimeZoneOffset = hx.preferences.timezone().replace(/^UTC\+?/,'').split(":").map (item, index) ->
+      if index is 0
+        Number(item) * 60
+      else
+        Number(item)
+    .reduce (current, previous) ->
+      current + previous
+    hexagonTimeZoneOffset
```

Comment:
This function can also be quite short. I can offer something like this if you prefer: 
```
result = /(.)(\d{2}):(\d{2})$/.exec("UTC-02:00")
if result 
  [sign,hoursPart,minutesPart] = result.slice(1)
  console.log sign, hoursPart, minutesPart
else 
  console.log "Cannot extract date parts."
```